Newsletters: add 346 (2025-03-21)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Client/Services @bitschmidty 
- [x] Notable merges @Gustavojfe 

I might also add something about the testnet3 discussion on the ML, but I see AntoineP just started a related thread, so I'm inclined to let that stew for a few days and summarize it next week.

LarryRuane commented on _posts/en/newsletters/2025-03-21-newsletter.md:
```diff
@@ -0,0 +1,106 @@
+  an external estimator) and ends with the total budget being allocated
+  to fees when the deadline is one block away."
+
+  He additionally describes how the new logic helps protected against
```
```suggestion
  He additionally describes how the new logic helps protect against
```

LarryRuane commented on _posts/en/newsletters/2025-03-21-newsletter.md:
```diff
@@ -0,0 +1,106 @@
+  pinning].
+
+  He concludes with a link-filled summary of several "LND-specific bug
+  and vulnerability fixes" made though the improved logic.  Abubakar
```
```suggestion
  and vulnerability fixes" made through the improved logic.  Abubakar
```

LarryRuane commented on _posts/en/newsletters/2025-03-21-newsletter.md:
```diff
@@ -0,0 +1,106 @@
+  He concludes with a link-filled summary of several "LND-specific bug
+  and vulnerability fixes" made though the improved logic.  Abubakar
+  Sadiq Ismail [replied][ismail sweep] with some suggestions for how all
+  LN implementations (in addition to other software) can make more
+  effective use of Bitcoin Core's fee estimation.  Several other
```
```suggestion
  LN implementations (in addition to other software) can more
  effectively use Bitcoin Core's fee estimation.  Several other
```

bitschmidty commented:
Pushed client/services section (ignore that I called it the stack exchange segment!)

bitschmidty commented on _posts/en/newsletters/2025-03-21-newsletter.md:
```diff
@@ -0,0 +1,227 @@
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #31649][] removes all checkpoint logic, replacing it with the
```
"replacing it with" in this context sounds like this PR is replacing it, but really presync was a few years ago. Id suggest some slight rewording here @Gustavojfe

harding commented:
Made edits for all feedback to my sections (thanks @LarryRuane @bitschmidty !), added lede, and added topic entries.

I deleted the releases/RCs section but then thought to check whether Bitcoin Core 29 RCs were available, which they are, so I'll add it back in an update later today.

Thanks everyone!  I'll check back for additional review comments around 01:00 UTC.

azuchi commented on _posts/en/newsletters/2025-03-21-newsletter.md:
```diff
@@ -0,0 +1,223 @@
+
+  Morehouse then describes how these two strategies are combined in
+  LND's new sweeper system: "[HTLC][topic htlc] claims with matching
+  deadlines [are aggregated] into a single [batched transaction][topic
```
> [are aggregated]

Is this a dead link?

jirijakes replied:
I guess these are externally inserted words into quote.

azuchi replied:
I see. thanks!

LarryRuane commented on _posts/en/newsletters/2025-03-21-newsletter.md:
```diff
@@ -0,0 +1,234 @@
+  calculated as the sum of the budgets for the individual HTLCs in the
+  transaction. Based on the transaction budget and deadline, a fee
+  function is computed that determines how much of the budget is spent
+  as the deadline approaches.  By default, a linear fee function is used
```
```suggestion
  as the deadline approaches.  By default, a linear fee function is used,
```

LarryRuane commented on _posts/en/newsletters/2025-03-21-newsletter.md:
```diff
@@ -0,0 +1,234 @@
+- [Bitcoin Core #31649][] removes all checkpoint logic, which is no longer
+  necessary following the headers presync step implemented years ago (see
+  Newsletter [#216][news216 presync]) that enables a node during Initial Block
+  Download (IBD) to determine if a chain of headers is valid by comparing its
```
```suggestion
  Download (IBD) to determine if a chain of headers may be valid by comparing its
```
Maybe, because otherwise it sounds like that's the only validity check that's needed.

LarryRuane commented on _posts/en/newsletters/2025-03-21-newsletter.md:
```diff
@@ -0,0 +1,234 @@
+
+- [Eclair #3037][] enhances the `listoffers` command (See Newsletter
+  [#345][news345 offers]) to return all relevant [offer][topic offers] data,
+  including the `createdAt` and `disabledAt` timestamps instead of just raw
```
```suggestion
  including the `createdAt` and `disabledAt` timestamps, instead of just raw
```
